emcc

```diff
@@ -975,11 +975,27 @@ try:
       logging.warning('not all asm.js optimizations are possible with --closure 2, disabling those - your code will be run more slowly')
       shared.Settings.ASM_JS = 2
 
-  if shared.Settings.POINTER_MASKING:
-    logging.warning('POINTER_MASKING is experimental')
-    size = shared.Settings.POINTER_MASKING + 1 # size plus 1 should be a power of 2
-    if size & (size-1) != 0:
-      raise Exception('POINTER_MASKING value must be a power of 2 minus 1')
+  if shared.Settings.POINTER_MASKING_DYNAMIC:
```

Comment:
Perhaps I wasn't clear enough. This is the code i meant in `emcc`. The DYNAMIC stuff would better off be inside the `elif shared.Settings.POINTER_MASKING:` (and that would be an if, not an elif).